app/models/flexible_page/flexible_section/base.rb

```diff
@@ -0,0 +1,11 @@
+module FlexiblePage::FlexibleSection
+  class Base
+    attr_reader :data, :flexible_page, :type
+
+    def initialize(flexible_section_hash, flexible_page)
```

Comment:
I was getting very confused about what `flexible_page` is here. But it's the content item, right? Perhaps then it should be called `content_item`?

Reply:
I'm assuming the whole content item is being passed in here so that the `links` hash among others, can still be accessed?